packages/forms/src/UIForm/fieldsets/CollapsibleFieldset/CollapsibleFieldset.component.js

```diff
@@ -12,74 +14,70 @@ function defaultTitle(_, schema) {
 }
 
 export default function createCollapsibleFieldset(title = defaultTitle) {
-	class CollapsibleFieldset extends React.Component {
-		constructor(props) {
-			super(props);
-			this.toggle = this.toggle.bind(this);
-		}
-
-		toggle(event) {
+	function CollapsibleFieldset(props) {
+		function toggle(event) {
 			event.stopPropagation();
 			event.preventDefault();
 			const payload = {
-				schema: this.props.schema,
+				schema: props.schema,
 				value: {
-					...this.props.value,
-					isClosed: !this.props.value.isClosed,
+					...props.value,
+					isClosed: !props.value.isClosed,
 				},
 			};
-			this.props.onChange(event, payload);
+			props.onChange(event, payload);
 		}
 
-		render() {
-			const { id, schema, value, ...restProps } = this.props;
-			const { items } = schema;
-			const iconTransform = !this.props.value.isClosed ? 'flip-vertical' : null;
+		const { id, schema, value, ...restProps } = props;
+		const { t } = useTranslation(I18N_DOMAIN_FORMS);
+		const { items } = schema;
+		const iconTransform = !props.value.isClosed ? 'flip-vertical' : null;
+		const expandLabel = t('FIELDSET_EXPAND', { defaultValue: 'Expand' });
+		const collapseLabel = t('FIELDSET_COLLAPSE', { defaultValue: 'Collapse' });
 
-			return (
-				<fieldset
-					className={classNames('form-group', theme['collapsible-panel'], 'collapsible-panel')}
+		return (
+			<fieldset
+				className={classNames('form-group', theme['collapsible-panel'], 'collapsible-panel')}
+			>
+				<div
+					onDoubleClick={toggle}
+					id={id && `${id}__title_bar`}
+					role="button"
+					className={theme['title-bar']}
 				>
-					<div
-						onDoubleClick={this.toggle}
-						id={id && `${id}__title_bar`}
+					<div // eslint-disable-line jsx-a11y/no-static-element-interactions
```

Comment:
div -> button

Reply:
I agree but I'd rather not touch the markup in this PR, especially since someone disabled this rule on purpose

Reply:
This is ok, I even think we can remove it, because the accessible way to toggle is the toggle button. This is just a additional way with mouse.
But this is out of this PR scope